rabbitmq-server.yaml

```diff
@@ -78,6 +78,51 @@ subpackages:
     pipeline:
       - uses: split/manpages
 
+  - name: rabbitmq-server-compat
+    description: Compat package for the Bitnami Rabbitmq Helm chart
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: rabbitmq
+          version-path: 3.13/debian-12
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/opt/bitnami/rabbitmq
+          ln -sf /usr/lib/rabbitmq/lib/rabbitmq_server-3.13.2/plugins ${{targets.contextdir}}/opt/bitnami/rabbitmq/
+          ln -sf /usr/lib/rabbitmq/lib/rabbitmq_server-3.13.2/escript ${{targets.contextdir}}/opt/bitnami/rabbitmq/
+          ln -sf /usr/lib/rabbitmq/lib/rabbitmq_server-3.13.2/sbin ${{targets.contextdir}}/opt/bitnami/rabbitmq/
```

Comment:
```suggestion
          ln -sf /usr/lib/rabbitmq/lib/rabbitmq_server-${{package.version}}/plugins ${{targets.contextdir}}/opt/bitnami/rabbitmq/
          ln -sf /usr/lib/rabbitmq/lib/rabbitmq_server-${{package.version}}/escript ${{targets.contextdir}}/opt/bitnami/rabbitmq/
          ln -sf /usr/lib/rabbitmq/lib/rabbitmq_server-${{package.version}}/sbin ${{targets.contextdir}}/opt/bitnami/rabbitmq/
```

Reply:
You're my hero @rawlingsj! Thanks for spotting that, been staring at these for so long